controllers/rest/controller_standard.go

```diff
@@ -664,3 +667,124 @@ func HealthCheck(api *RESTAPI, projection projections.Projection, commandDispatc
 	}
 
 }
+
+//AuthorizationMiddleware handling JWT in incoming Authorization header
+func AuthorizationMiddleware(api *RESTAPI, projection projections.Projection, commandDispatcher model.CommandDispatcher, eventSource model.EventRepository, entityFactory model.EntityFactory, path *openapi3.PathItem, operation *openapi3.Operation) echo.MiddlewareFunc {
+	var openIdConnectUrl string
+	securityCheck := true
+	var verifiers []*oidc.IDTokenVerifier
+	algs := []string{"RS256", "RS384", "RS512", "HS256"}
+	if operation.Security != nil && len(*operation.Security) == 0 {
+		securityCheck = false
+	}
+	for _, schemes := range api.Swagger.Components.SecuritySchemes {
+		//get the open id connect url
+		if openIdUrl, ok := schemes.Value.ExtensionProps.Extensions[OPENIDCONNECTURLEXTENSION]; ok {
+			err := json.Unmarshal(openIdUrl.(json.RawMessage), &openIdConnectUrl)
+			if err != nil {
+				api.EchoInstance().Logger.Errorf("unable to unmarshal open id connect url '%s'", err)
+			} else {
+				//check if it is a valid open id connect url
+				if !strings.Contains(openIdConnectUrl, ".well-known/openid-configuration") {
```

Comment:
the name of the endpoint doesn't make it a valid open id connect URL. It's if you call the URL and it has the jwks_uri in the json response that is returned